getting-started/run-queries-from-console.md

```diff
@@ -85,9 +122,9 @@ Now you have set up PopSQL, see how to easily do the following:
 
 
 
-## FAQ
+### FAQ
```

Comment:
This is a bit confusing because before this page was only about PopSQL, but now it's about SQL editor **plus** PopSQL, but the FAQ is still **only** about PopSQL. For example, the VPC answer is purely about PopSQL, and doesn't address the SQL editor + VPC